spec/models/item_spec.rb

```diff
@@ -0,0 +1,30 @@
+require 'rails_helper'
+
+RSpec.describe Item, type: :model do
+  describe 'relationships' do
+    it { should belong_to(:merchant) }
+    it { should have_many(:invoice_items) }
+    it { should have_many(:invoices).through(:invoice_items) }
+  end
+
+  describe 'validations' do
+    it { should validate_presence_of(:name) }
+    it { should validate_presence_of(:description) }
+    it { should validate_presence_of(:unit_price) }
```

Comment:
Add validation test for numericality: `it { should validate_numericality_of(:unit_price) }`